File: 01-Variables and Arrays/Main.py
import numpy as np
#1
A=1
#2
B=np.uint8(1)
#3
C=np.float64(1)
#4
D=round(1.5)
#5
E=np.zeros((5,5))
#6 
F=np.ones((5,5))
#7 
G=np.random.rand(5,5)
#8 
H=np.random.randint(1,5,(3,3))
#9 
I=np.arange(1,6)
#10 
J=np.flip(np.arange(1,6),0)
#11 
K=np.transpose(np.ones((3,4)))
#12 
L=np.reshape(np.ones((5,5)),[25,1])
#13 
M=np.ones((5,5)); M=np.reshape(M,(1,-1))
#14
N=np.linspace(1,10,5)
#15
O=np.concatenate(([1,2],[3,4,5]),axis=0)
#16
P=np.random.rand(5,5)
np.save('Var.npy',P)
#17
P=np.load('Var.npy')
#18 struct
class struct:
        pass
R=struct()
R.var1=12
R.var2=13
    

File: 01-Variables and Arrays/Main.m
%1
A=1
%2
B=uint8(1)                    
%3
C=double(1)                 
%4
D=round(1.5)
%5
E=zeros(5,5)
%6 
F=ones(5,5)
%7 
G=rand(5,5)
%8 
H=randi([1,5],3,3)
%9 
I=[1:5]
%10 
J=flip([1:5],2)
%11 
K=ones(3,4)'
%12 
L=reshape(ones(5,5),[25,1])
%13 
M=ones(5,5); M=M(:)
%14
N=linspace(1,10,5)
%15
O=cat(2,[1,2],[3,4,5])
%16
P=rand(5,5)
save('Var.mat','P')
%17
load('Var.mat')
%18 struct
R.var1=12
R.var2=13









